swagger.yml

```diff
@@ -1970,14 +2001,70 @@ paths:
           required: true
           type: integer
       responses:
-        '200':
+        "200":
           description: successful operation
           schema:
             type: array
             items:
-              $ref: '#/definitions/PortalPlayer'
-        '400':
+              $ref: "#/definitions/PortalPlayer"
+        "400":
           description: error
+  /player/active:
+    get:
+      tags:
+        - players
+      summary: Get all active players
+      description: All active players
+      operationId: getActivePlayers
+      produces:
+        - application/json
+      responses:
+        "200":
+          description: successful operation
+          schema:
+            type: array
+        "500":
+          description: error, unknown origin
+    /player/active/powerfive:
+      get:
+        tags:
+          - players
+        summary: Get all active power five players
+        description: All active power five players
+        operationId: getActivePowerFivePlayers
+        produces:
+          - application/json
+        responses:
+          "200":
+            description: successful operation
+            schema:
+              type: array
+          "500":
+            description: error, unknown origin
+  /player/position:
+    get:
+      tags:
+        - players
+      summary: Get all active players at a given position
+      description: All active players at a given position
+      operationId: getActivePlayersByPosition
+      produces:
+        - application/json
+      parameters:
+        - name: positionId

```

Comment:
This should go off of position abbreviation to remain consistent with the rest of the API.